packages/ketchup/src/components/kup-data-table/kup-data-table-declarations.ts

```diff
@@ -114,6 +120,7 @@ export interface Column {
         p: string;
         k: string;
     };
+    objs?: Obj[];
```

Comment:
Should be used `KupObj` from kupobjects-declarations instead of `Obj`. Also "objs" is not very clear, which objs does it contain?